packages/dds/tree/api-report/tree.api.md

```diff
@@ -1083,7 +1083,7 @@ export interface ITransaction {
 }
 
 // @public
-export interface ITree extends IChannel {
+export interface ITree extends IFluidLoadable {
```

Comment:
This is the main change. The rest it just related cleanup to avoid this causing issues.